docs/installation_manual_in_cluster.md

```diff
@@ -76,7 +77,7 @@ If you built your own Docker image for Preflight, you need to edit `deployment/k
 We also need to customize some things in the configuration file. Edit `deployment/kubernetes/overlays/scanner/config/preflight.yaml` and chagne:
```

Comment:
```suggestion
We also need to customize some things in the configuration file. Edit `deployment/kubernetes/overlays/scanner/config/preflight.yaml` and change:
```
Just noticed this typo, not related to any change in this PR but would be a place time to fix it.